spark/dl/src/main/scala/com/intel/analytics/bigdl/models/lenet/LeNet5.scala

```diff
@@ -103,4 +104,22 @@ object LeNet5 {
       .add(mkldnn.ReorderMemory(mkldnn.HeapData(outputShape, Memory.Format.nc)))
     model
   }
+
+  def dnnGraph(batchSize: Int, classNum: Int): mkldnn.DnnGraph = {
```

Comment:
How about add an option to example?

Reply:
ok. How about using "graph" & "mkldnn" engine type to call dnnGraph?

Reply:
yeah, good idea. Changed